utils/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Need to make this change to fix this error.
ref: https://javadoc.io/doc/org.mockito/mockito-core/3.12.4/org/mockito/Mockito.html#Mocking_Final
```
org.mockito.exceptions.base.MockitoException: 
The used MockMaker PowerMockMaker does not support the creation of static mocks

Mockito's inline mock maker supports static mocks based on the Instrumentation API.
You can simply enable this mock mode, by placing the 'mockito-inline' artifact where you are currently using 'mockito-core'.
Note that Mockito's inline mock maker is not supported on Android.
```

Reply:
We can add it to the excluded files in pom.xml as well @vishesh92 (if that is the wiser thing to do?)

Reply:
I don't have much experience with `pom.xml`. Are you suggesting that we add exclusion criteria to `powermock` dependencies in `pom.xml`

Reply:
no , just an exclusion to the license requitrement for this config file that only contains code that we did not come up with ;)
in pom.xml at project level around line 1064 add
```
                            <exclude>utils/src/test/resources/mockito-extension/org.mockito.plugins.MockMaker</exclude>
```

Reply:
Got it. I have updated the PR.